### DISCUSSION TITLE
cs_tools tools git branches deploy getting an error

### DISCUSSION THREAD

**Discussion by RandyHoffpauir** [thoughtspot/cs_tools#183] [General]
.\cs_tools.exe tools git branches deploy --branch-name releases/myreleasebranch --deploy-type DELTA --deploy-policy ALL_OR_NONE

Here is a snippet of the log file:

           DEBUG    Missing privilege                     datastructures.py:190
                    'THIRDPARTY_ANALYSIS' from CS Tools,                       
                    please contact us to update it                             
                    https://github.com/thoughtspot/cs_too                      
                    ls/discussions/                                            
##[error]C:\Users\VssAdministrator\AppData\Roaming\cs_tools\.cs_tools\Lib\site-packages\pydantic\main.py:390: UserWarning: Pydantic serializer warnings:
  Expected `datetime` but got `str` with value `'2024-10-29T18:42:50.683070Z'` - serialized value may not be as expected
  return self.__pydantic_serializer__.to_python(

**Comment by boonhapus** [comment]
Thanks @RandyHoffpauir ! I'll add the missing privilege and silence the pydantic warning in the next release. Nothing looks like it's failing here though which is good.
